include/inviwo/core/util/defaultvalues.h

```diff
@@ -74,6 +74,21 @@ struct InviwoDefaults<unsigned char> {
     }
 };
 template <>
+struct InviwoDefaults<short> {
+    static constexpr auto get() {
+        return InviwoDefaultData{StaticString{"Int16"}, uvec2(1, 1), short{0},
```

Comment:
I think "Short" and "UShort" is more inline with the other names? (If I would have done it from scratch I would have named them all "IntX")